src/main/java/org/aszjch/demoapp/domain/ArticleServiceImpl.java

```diff
@@ -12,24 +12,36 @@
 @RequiredArgsConstructor
 public class ArticleServiceImpl implements ArticleService {
 
-  private final ArticleRepository repository;
-  private final ArticlePublisher publisher;
-
-  @Override
-  public List<Article> get() {
-    return repository.findAll();
-  }
-
-  @Override
-  public Article create(Article article) {
-    Article saved = repository.save(article);
-    publisher.publish("Article created: " + saved.getId()); // todo: probably to be deleted, just an example
-    return saved;
-  }
-
-  @Override
-  public Optional<Article> getById(Long id) {
-    return repository.findById(id);
-  }
+    private final ArticleRepository repository;
+    private final ArticlePublisher publisher;
+
+    @Override
+    public List<Article> get() {
+        return repository.findAll();
+    }
+
+    @Override
+    public Article create(Article article) {
+        Article saved = repository.save(article);
+        publisher.publish("Article created: " + saved.getId()); // todo: to be expanded once article has more data
```

Comment:
jak rozszerzymy obiekt, to wyślijmy coś fajnego

Reply:
Ogarnę to na kolejnym branchu, trochę za bardzo się rozpędziłem z dodawaniem logiki wokół ArticleFile...